pkg/resources/masking_policy.go

```diff
@@ -106,27 +109,37 @@ var maskingPolicySchema = map[string]*schema.Schema{
 		Optional:    true,
 		Description: "Specifies a comment for the masking policy.",
 	},
-	"qualified_name": {
-		Type:        schema.TypeString,
-		Computed:    true,
-		Description: "Specifies the qualified identifier for the masking policy.",
-	},
+	FullyQualifiedNameAttributeName: schemas.FullyQualifiedNameSchema,
```

Comment:
I'm wondering state upgrader is needed here. They're computed fields that "should" be re-calculated every read which means `fully_qualified_name` will be populated after the initial read (after upgrade) and sadly state upgrader won't help with changing the name from `qualified_name` to `fully_qualified_name`.

Reply:
Removed.